app/views/subscription/add.phtml

```diff
@@ -218,4 +218,35 @@
 			</div>
 		</div>
 	</form>
+
+	<h2>
+		<?= _t('sub.title.add_dynamic_opml') ?>
+		<span title="<?= _t('sub.category.dynamic_opml') ?>">🗲</span>
```

Comment:
I am not so happy to have here an icon. It would be the first time to have an icon in the headline

Reply:
Actually, my intention is to have that icon not only in the title, but after the name of a dynamic category everywhere it is used, since it is a behaviour that is so peculiar that it can lead to confusion when forgotten:

![image](https://user-images.githubusercontent.com/1008324/174403671-5ef9e631-7ee3-4abf-ab99-3a3c3d20bda9.png)

![image](https://user-images.githubusercontent.com/1008324/174403722-450ee1c4-5082-42d5-801b-76a7269e731a.png)

Reply:
I do not see the benefit right now, because the update of feed list needs an manual update.

Maybe it would make (more) sense to mark the feeds that are imported with dynamic OPML? These feeds could disappear after a feed list update.

Reply:
The feeds under a dynamic category will automatically change over time. It is the category that makes it special, not the feeds, which can be for instance moved to another *normal* category

Reply:
(I am not sure I have fully understood your comment)